src/components/home/posts/PCard.module.css

```diff
@@ -0,0 +1,9 @@
+.posts-card{
+    min-width: 309px;
+    min-height: 131px;
+    width: 15vw;
+    height: 10vh;
+    border: 1px solid #D2D2D2;
+    background-color: #fff !important;
+    padding: 10px;
+}
```

Comment:
We should use all dynamic units not px for sizing. Or make use of breakpoints and @media query